src/components/vouchers/VoucherRedeem.tsx

```diff
@@ -0,0 +1,298 @@
+                    titleStyle={styles.progressTitle}
+                    value={((defaultTime - remainingTime) / defaultTime) * 100}
+                  />
+                </Wrapper>
+
+                <Wrapper>
+                  <Touchable
+                    onPress={() => {
+                      setIsVisible(false);
+                      setIsRedeemVoucher(false);
+                      setIsChecked(false);
+                    }}
+                  >
+                    <BoldText small center underline lightest>
+                      {texts.voucher.detailScreen.close}
+                    </BoldText>
+                  </Touchable>
+                </Wrapper>
+              </>
+            ) : (
+              <>
+                <Wrapper>
+                  <BoldText lightest>{texts.voucher.detailScreen.sheetTitle}</BoldText>
+                </Wrapper>
+
+                <Wrapper style={styles.noPaddingTop}>
+                  <RegularText lightest>{texts.voucher.detailScreen.sheetDescription}</RegularText>
+                </Wrapper>
+
+                <Wrapper style={styles.noPaddingTop}>
+                  <Checkbox
+                    checked={isChecked}
+                    checkedColor={colors.surface}
+                    checkedIcon="check-square-o"
+                    containerStyle={styles.checkbox}
+                    title={texts.voucher.detailScreen.checkboxLabel}
+                    onPress={() => setIsChecked(!isChecked)}
+                    uncheckedColor={colors.surface}
+                    uncheckedIcon="square-o"
+                    lightest
+                  />
+                </Wrapper>
+
+                {!!maxPerPerson && maxPerPerson > 1 && (
+                  <Wrapper style={styles.noPaddingTop}>
+                    <View style={styles.quantityContainer}>
+                      <RegularText lightest small>
+                        {texts.voucher.detailScreen.desiredQuantity}:
+                      </RegularText>
+
+                      <View style={styles.quantityButtonContainer}>
+                        <TouchableOpacity
+                          style={styles.quantityButton}
+                          onPress={() => {
+                            if (quantity > 1) {
+                              setQuantity(quantity - 1);
+                            }
+                          }}
+                        >
+                          <BoldText lightest>-</BoldText>
+                        </TouchableOpacity>
+                        <BoldText lightest>{quantity}</BoldText>
+                        <TouchableOpacity
+                          style={styles.quantityButton}
+                          onPress={() => {
+                            if (quantity < maxPerPerson) {
+                              setQuantity(quantity + 1);
+                            }
+                          }}
+                        >
+                          <BoldText lightest>+</BoldText>
```

Comment:
please use the "fullwidth" chats for - and plus:

* －
* ＋
* 
that looks more in place
<img width="275" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/4a5be766-bd37-4f4d-ac0f-cb2962972cc4">

Reply:
I'm sorry, but I don't know exactly what you mean.

Reply:
change the chars with the ones i commented. they are different, a bit larger. and as you see in the screenshot, it looks more in place.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/989/commits/4b24452b532b5b637b88c8df649fb1318128497a